Sources/TuistKit/Utils/ImportSourceCodeScanner.swift

```diff
@@ -5,65 +5,85 @@ enum ProgrammingLanguage {
     case objc
 }
 
+struct Match {
```

Comment:
```suggestion
private struct Match {
```

This is an implementation detail that shouldn't spill out to the rest of the `TuistKit` module